keyboards/zlant/keymaps/default/keymap.c

```diff
@@ -0,0 +1,101 @@
+#include "zlant.h"
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	KEYMAP(
```

Comment:
Please replace all of the `KEYMAP`s with `LAYOUT`. 

If this fits one of these layouts, let me know: https://github.com/qmk/qmk_firmware/tree/master/layouts/default

Reply:
Also, you may want to add defines for your layers, so it's easier for reference. 

Like this: 
https://github.com/qmk/qmk_firmware/blob/38f204db3031475aef2252e1ad11e4c87fb559f1/layouts/community/60_ansi/mechmerlin-ansi/keymap.c#L3-L5

But you'd need to add a bit before each `KEYMAP`/`LAYOUT`: `[NAME] = `